pygments/styles/dracula.py

```diff
@@ -0,0 +1,107 @@
+"""
+    pygments.styles.dracula
+    ~~~~~~~~~~~~~~~~~~~~~
+
+    Pygments version of `Dracula` from https://github.com/dracula/dracula-theme.
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+
+    Based on the Dracula Theme for pygments by Chris Bracco.
+    See https://github.com/dracula/pygments/tree/fee9ed5613d1086bc01b9d0a5a0e9867a009f571
+"""
+
+from pygments.style import Style
+from pygments.token import Keyword, Name, Comment, String, Error, \
+    Literal, Number, Operator, Other, Punctuation, Text, Generic, \
+    Whitespace
+
+class DraculaStyle(Style):
+
+    background_color = "#282a36"
+    default_style = ""
+
```

Comment:
Can you please add the various `linenos` styles and the `highlight` color as well?